overlay/make-package-set/full.nix

```diff
@@ -11,6 +11,7 @@
   packageFun,
   workspaceSrc ? null,
   rustChannel,
+  target,
```

Comment:
@trha or @ebkalderon would be better to review this, but the only thing I notice is that you're not setting a default? 

Seems like this should be `target ? (rustBuilder.rustLib.realHostTriple stdenv.targetPlatform)` or something like that, so that the original behaviour is maintained.

Reply:
The default is handled further upstream in `simplified.nix`